mbuild/utils/io.py

```diff
@@ -76,14 +88,14 @@ class DelayImportError(ImportError, SkipTest):
 # pip install foyer
 '''
 
-MESSAGES['protobuf'] = ''' 
+MESSAGES['protobuf'] = '''
 The code at {filename}:{line_number} requires the "protobuf" package
 
 protobuf can be installed using:
 
 # conda install -c anaconda protobuf
```

Comment:
Unrelated to this PR, but you should recommend `conda-forge` over any other channels when you can, since `mbuild` is provided through that channel. Clashes between channel versions are rough.
```suggestion
# conda install -c conda-forge protobuf
```

Reply:
Also, GitHub won't let me add a comment on this code since it's far from your diff, but `gsd` is also available via `pip` for all platforms. Given that, I believe all of the messages are identical except for the package name, and you might be able to refactor all the messages into one template.

Reply:
I am in favor of this change (fine to do in another PR). 2-3 years ago, when this function was being built up, dependencies were all over the place, each package in a different conda. Now there are effectively only 2 places to get ~90% of packages

Reply:
Agreed, ill put that change in a separate PR, thanks for the catch!